src/sentry/static/sentry/app/views/organizationHealth/eventsTableChart.jsx

```diff
@@ -87,13 +97,29 @@ class EventsTableChart extends React.Component {
                 {items[3]}
               </Flex>
             </PercentageContainer>
-          </React.Fragment>
+          </Row>
         )}
       />
     );
   }
 }
 
+const Row = styled(function RowComponent({className, data, rowIndex, onClick, children}) {
+  return (
+    <div
+      className={className}
+      onClick={e =>
+        typeof onClick === 'function' && rowIndex > -1 && onClick(data[rowIndex], e)}
```

Comment:
How could a rowIndex ever be -1? I assume that this component would never be rendered in that case